index.html.in

```diff
@@ -225,7 +225,7 @@
                                                 <span data-text="Translate_Webpage">Translate a webpage</span>
                                             </button>
                                         </div>
-                                        <div class="col-lg-3 col-md-6  pull-right">
+                                        <div class="col-lg-3 col-md-6  pull-right" id="translateOptionsContainer">
```

Comment:
Mind fixing the extra space here? Also, I think we could just push `#translateOptions` up? I checked the JS and it seems safe.